src/fluree/server/consensus/handlers/tx_queue.clj

```diff
@@ -17,12 +15,12 @@
   ;; This map is created in the 'transact' http handler via
   ;; fluree.server.consensus.core/queue-new-transaction
 
-  {:txn       txn
-   :conn-type conn-type
-   :size      (count txn)
-   :tx-id     tx-id
-   :ledger-id ledger-id
-   :instant   (System/currentTimeMillis)})
+  '{:txn       txn
```

Comment:
should this map really be quoted? there are a bunch of function calls inside.

Reply:
It's quoted because most of the symbols inside are not resolvable, which clj-kondo flags. It's just a commented-out example anyway.